opm/parser/eclipse/EclipseState/Schedule/Completion.cpp

```diff
@@ -179,7 +182,8 @@ namespace Opm {
         const WellCompletion::DirectionEnum direction = WellCompletion::DirectionEnumFromString(compdatRecord.getItem("DIR").getTrimmedString(0));
 
         for (int k = K1; k <= K2; k++) {
-            CompletionPtr completion(new Completion(I , J , k , state , connectionTransmissibilityFactor, diameter, skinFactor, direction ));
+            double depth = grid.getCellDepth( I,J,k );
+            CompletionPtr completion(new Completion(I , J , k , depth , state , connectionTransmissibilityFactor, diameter, skinFactor, direction ));
```

Comment:
TODO: remove this vector-of-pointers.

(not for this task, just making a note of it).